client/src/components/timetable/MidsemSched.jsx

```diff
@@ -27,4 +29,10 @@ const MidsemSched = props => {
   );
 };
 
-export default MidsemSched;
+const mapStateToProps = state => {
+  return {
+    myCourses: state.updateTT.myCourses
+  };
+};
+
```

Comment:
Is `myCourses` being changed here? If not, do we need the `mapStateToProps` here?

Reply:
nope